rotkehlchen/db/filtering.py

```diff
@@ -989,6 +990,14 @@ def make(
                 asset_key='asset',
                 operator='NOT IN',
             ))
+        if amounts is not None:
+            str_amounts = [str(amount) for amount in amounts]
+            filters.append(DBMultiStringFilter(
+                and_op=True,
+                column='amount',
+                values=str_amounts,
+                operator='IN',
+            ))
```

Comment:
Something minor but you can just do

```suggestion
            filters.append(DBMultiStringFilter(
                and_op=True,
                column='amount',
                values=[str(amount) for amount in amounts],
                operator='IN',
            ))
```